### PR TITLE
Generate certs for registry.ci.openshift.org

### PR DESCRIPTION
**Pull request by hongkailiu**
This PR starts the work on https://issues.redhat.com/browse/DPTP-1559

/cc @stevekuznetsov @alvaroaleman

### CODE REVIEW

**Comment by hongkailiu** [comment]
Tried already with manual application: It is ready to be used.

```
oc get certificate -n openshift-image-registry
NAME             READY   SECRET             AGE
registry-appci   True    public-route-tls   97m
```

File: clusters/app.ci/cert-manager/registry-appci_certificate.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  labels:
+    gcp-project: openshift-ci-infra
+  name: registry-appci
```
Comment (alvaroaleman) [inline clusters/app.ci/cert-manager/registry-appci_certificate.yaml:6]: I think the `appci` is implicit by the fact that this is on the `app,ci` cluster, maybe `registry-ci-openshift-org`?

Comment (hongkailiu) [reply clusters/app.ci/cert-manager/registry-appci_certificate.yaml]: Updated.

File: clusters/app.ci/cert-manager/registry-ci-openshift-org_certificate.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  labels:
+    gcp-project: openshift-ci-infra
+  name: registry-ci-openshift-org
+  namespace: openshift-image-registry
+spec:
+  secretName: public-route-tls
```
Comment (alvaroaleman) [inline clusters/app.ci/cert-manager/registry-ci-openshift-org_certificate.yaml:9]: Maybe name it like the cert but with a `-tls` suffix? The build-in route is also public

File: clusters/app.ci/cert-manager/registry-ci-openshift-org_certificate.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  labels:
+    gcp-project: openshift-ci-infra
+  name: registry-ci-openshift-org
+  namespace: openshift-image-registry
+spec:
+  secretName: customized-route-tls
```
Comment (alvaroaleman) [inline clusters/app.ci/cert-manager/registry-ci-openshift-org_certificate.yaml:9]: Lets please just name it like the cert, with a `-tls` suffix. `customized-route` doesn't really make anything clear either

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**